packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -350,7 +350,7 @@ export async function deleteDatasetPrompt(
                             : ` ${currNode.getLabel()}`;
                         nodesDeleted.push(deleteItemName);
                     } catch (err) {
-                        // do nothing; delete next
+                        this.log.error(err);
```

Comment:
In this file I think it should be: globals.LOG.error(err). Can you please verify? I just gave you an example. 😆